src/main/java/frc/robot/commands/AutonomousTools/VisionCubeTracker.java

```diff
@@ -0,0 +1,169 @@
+package frc.robot.commands.AutonomousTools;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import frc.robot.*;
+
+@SuppressWarnings("deprecation")
+public class VisionCubeTracker extends Command{
```

Comment:
This is a "cube" tracker from last season... Does this still pertain to Deep Space?